astropy/coordinates/angle_utilities.py

```diff
@@ -753,9 +753,6 @@ def offset_by(lon, lat, posang, distance):
         The position of the final point.  If any of the angles are arrays,
         these will contain arrays following the appropriate `numpy` broadcasting rules.
         0 <= lon < 2pi.
-
-    Notes
```

Comment:
This might be intentional. Something there is docstring magic that goes on somewhere else. Maybe @adrn or @mhvk knows.

Reply:
This comment also applies to all other similar changes.

Reply:
That just seems a copy & paste mistake... Or maybe the talk about the way the offset was done was meant to be added. For this clean-up PR, it makes sense to just delete it!

Note that the two in `solar_system.py` should be kept, though!